pkg/reconciler/awssnssource/client_test.go

```diff
@@ -0,0 +1,198 @@
+/*
+Copyright (c) 2020 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package awssnssource
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+
+	"github.com/triggermesh/aws-event-sources/pkg/apis/sources/v1alpha1"
+)
+
+func TestAwsCredentials(t *testing.T) {
+	const (
+		ns = "fake-namespace"
+
+		accessKeyIDKey     = "key-id"
+		accessKeyIDVal     = "fake key ID"
+		secretAccessKeyKey = "secret-key"
+		secretAccessKeyVal = "fake secret"
+	)
```

Comment:
This large test was simply moved from `subscription_test.go` to `client_test.go`, it's not new code.